packages/popover/src/Popover/Popover.spec.tsx

```diff
@@ -55,15 +55,18 @@ describe('packages/popover', () => {
     expect(ref.current).toBeDefined();
   });
 
-  test('accepts a portalRef', () => {
+  test('accepts a portalRef', async () => {
     const portalRef = createRef<HTMLElement>();
-    render(
-      <Popover portalRef={portalRef} data-testid="popover-test-id">
-        Popover Content
-      </Popover>,
-    );
+    waitFor(() => {
```

Comment:
Do we need to call `waitFor` here? I think `render` does that internally

Reply:
without the `waitFor` wrapping, `portalRef.Current` wasn't showing up in the document. It takes too long to render before the expect statement runs due to the `useIsomorphicLayoutEffect` in the portal component